std/experimental/allocator/building_blocks/region.d

```diff
@@ -1114,6 +1113,24 @@ nothrow @nogc:
         return false;
     }
 
+    /**
+    Deallocates all memory allocated by this region, which can be subsequently
+    reused for new allocations.
+    */
+    bool deallocateAll() @trusted pure nothrow @nogc
```

Comment:
This should not be `@trusted`. Deallocation is a `@system` operation from the pov of the allocator. The user can make it trusted through his own mechanisms, such as reference counting or other external meta data that can guarantee there are no more references to the allocated memory.

Reply:
I've just noticed the same is true for `Region.deallocateAll`. Could you remove `@safe` from there as well?
@nordlow I hope this change won't break your code.

Reply:
Good observation, thanks!